snippets/objectscript-class.json

```diff
@@ -84,7 +84,7 @@
 		"description": "Business Service Definition"
 	},
 	"BusinessOperation": {
-		"prefix":["BusinessOperation","Interoperability"],
+		"prefix":["BusinessOperation","Interoperability","ClassService"],
```

Comment:
`ClassOperation`?

Reply:
Thanks for this review, you are right, PR updated